part1/chapter6/quiz1/questions/question_2.rb

```diff
@@ -20,3 +20,33 @@
 # * Note: You can assume the players will only ever input `rock`,
 #   `paper` or `scissors`.
 
+puts "Player 1 pick rock, paper or scissors"
+ans1 = gets.chomp
+puts "Player 2 pick rock, paper or scissors"
+ans2 = gets.chomp
+
+if (ans1 == "rock")
+  if (ans2 == "rock")
```

Comment:
An if followed by an if could make use of an &&.

eg `if (ans1 == "rock" && ans2 == "rock" )